Medicine.h

```diff
@@ -0,0 +1,42 @@
+
+//kkk
+#include <string>
+#include <chrono>
+using namespace std;
+
+class Medicine {
+private:
+    int expirationYear;
+    int expirationMonth;
+    int expirationDay;
+    //std::chrono::time_point<std::chrono::system_clock> expDate;
+    int quantity;
+    int price;
+
+public:
+    string name;
+    Medicine() = default;
+
+
+    Medicine(string name, int price){
+        this->name = name;
+        this->price = price;
+
```

Comment:
Why use so many empty strings? 
Why don't you write "}" in the next string after end of constructor?